publication-commons/src/main/java/no/unit/nva/publication/permissions/ticket/grant/TransferTicketGrantStrategy.java

```diff
@@ -17,8 +18,8 @@ public TransferTicketGrantStrategy(TicketEntry ticket, UserInstance userInstance
 
     @Override
     public boolean allowsAction(TicketOperation permission) {
-        if (permission.equals(TRANSFER)) {
-            return canManageTicket() && userBelongsToReceivingTopLevelOrg() && anyValidReceivers();
+        if (permission.equals(TRANSFER) && ticket instanceof FilesApprovalEntry) {
+            return canManagePublishingTicket() && anyValidReceivers();
         }
 
         return false;
```

Comment:
Just a subjective suggestion. I find it nice to guard the method with operation type.

```suggestion
        if (!permission.equals(TRANSFER)) {
            return false;
        }

        return ticket instanceof FilesApprovalEntry && canManagePublishingTicket() && anyValidReceivers();
```